### ISSUE TITLE
U+0361 COMBINING DOUBLE INVERTED BREVE is misaligned when using Linguistics Pro font

### ISSUE THREAD

**Issue by ketsuban** [MiKTeX/miktex#138]
I'm using MiKTeX 2.9 on Windows 10.

I'm reporting this here in the hopes someone more knowledgeable than me can identify whether this is a XeTeX bug (and escalate it to the developers) or a bug in the font (in which case I can go to the creator).

When I generate a document using XeTeX and the Linguistics Pro font, the character U+0361 COMBINING DOUBLE INVERTED BREVE is incorrectly aligned such that the *right* half overlaps the character it is combined with. This [does not occur elsewhere](https://user-images.githubusercontent.com/5253816/40757220-64fce612-647f-11e8-87fe-08f3919784c6.png). Here is a minimal demonstration, for which you'll need to [download the font](https://fontlibrary.org/en/font/linguistics-pro) and put the font files in the same directory as the source file.

```latex
\documentclass{article}

\usepackage{fontspec}
\setmainfont{LinguisticsPro}[
    Extension = .otf,
    UprightFont = *-Regular,
]

\begin{document}
ad͡ʒa
\end{document}
```

**Comment by edocevoli** [comment]
It would be better to report this issue to the XeTeX developers. That said, I don't believe it is a MiKTeX-specific bug.

**Comment by u-fischer** [comment]
I see a difference between texlive and miktex:

miktex: 
![image](https://user-images.githubusercontent.com/4047173/40944452-1e189492-6855-11e8-94e1-3204decf76b7.png)

texlive:
![image](https://user-images.githubusercontent.com/4047173/40944473-3ac975c0-6855-11e8-89ff-16635a2d6e01.png)


My guess would be that it is due to the different libraries, harfbuzz e.g. is clearly older in miktex:

```
compiled with freetype2 version 2.8.1; using 2.8.1
compiled with graphite2 version 1.3.10; using 1.3.10
compiled with harfbuzz version 1.6.3; using 1.6.3
compiled with icu version 60.1; using 60.1
```

when compared with the version from a texlive xetex:

```
Compiled with ICU version 61.1; using 61.1
Compiled with FreeType2 version 2.9.1; using 2.9.1
Compiled with Graphite2 version 1.3.11; using 1.3.11
Compiled with HarfBuzz version 1.7.6; using 1.7.6
```

**Comment by edocevoli** [comment]
I have upgraded HarfBuzz to version 1,8,0 and this seems to fix the issue. The fix will be available with the [2.9.6730 milestone](https://github.com/MiKTeX/miktex/milestone/6).
